### ISSUE TITLE
implement RBAC using V1 method

### ISSUE THREAD

**Issue by wouter1975** [workfloworchestrator/orchestrator-ui-library#460]
Implement the same `allowed` RBAC implementation as currently used in V1.

Please consult Peter for some background instruction before starting.

from the readme:

## Actions and page views depending on who you are.

The orchestrator client can be configured to allow access to a page or to buttons in the client. The react app
consumes a webassembly OPA policy and evaluates if the user has the correct claims to view certain resources.
The function that gets called is the `allowed()` function in `src/utils/policy.ts`. The implementation of `allowed()`
is done as follows:

```typescript jsx
{allowed("/orchestrator/arbitrary/resource/name") && (
    <EuiFlexItem>
        ...
    </EuiFlexItem>
)}
```

Basically it boils down to: when a call to `allowed` returns true the component will be shown.

**When no policy is found, the orchestrator-client will allow access to the resource.
Any real access must be enforced by the API. The client only disables features with the `allowed` function**

Tasks:

- [x] Update login flow to go through OIDC proxy (configurable endpoint)
- [x] Expose session data to the client
- [x] Load policy on the `app` side -- add to config object
- [x] Introduce isAllowed function/hook in the library that is implemented in the app side
- [x] Bonus: Create component that uses the hook and conditionally renders the children: `<WfoRenderByPolicy>`
~- [ ] Verify that all policies are available in the dev environment, else create a story~ --> Out of scope, move to backend backlog
- [ ] Implement all policy checks from the V1 version of the GUI
